src/angular/projects/spark-extras-angular-dictionary/ng-package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "$schema": "../../node_modules/ng-packagr/ng-package.schema.json",
```

Comment:
The schema points to /node_modules/ng-packagr which we will be deleting in this PR.

Reply:
i know that angular-cli uses ng-packagr internally, so its ok that it points to the schema file there. Its a generated file when you do ng generate library.